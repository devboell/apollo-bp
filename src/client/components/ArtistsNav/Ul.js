import styled from 'styled-components';

const Ul = styled.ul`
  flex-basis: 30%;
  list-style: none;
  max-height: 30em;
  overflow-y: auto;
`;

export default Ul;
