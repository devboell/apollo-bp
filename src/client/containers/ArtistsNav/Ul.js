import styled from 'styled-components'

const Ul = styled.ul`
  list-style: none;
  overflow-y: auto;
`

export default Ul
