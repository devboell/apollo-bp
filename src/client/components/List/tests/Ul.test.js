import React from 'react'
import renderer from 'react-test-renderer'
import Ul from '../Ul'

it('Ul component snapshot', () => {
  const tree = renderer.create(<Ul/>).toJSON()
  expect(tree).toMatchSnapshot()
});
