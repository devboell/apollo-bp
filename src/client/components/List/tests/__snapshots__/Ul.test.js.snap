// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ul component snapshot 1`] = `
<ul
  className="ihkFxl"
/>
`;
